appinfo/install.php

```diff
@@ -1,11 +1,4 @@
 <?php
 
-$app = new \OCA\Files_Antivirus\AppInfo\Application();
```

Comment:
@PVince81 moved into migration.
install script is triggered before migrations, thus here we were trying to insert default rules into the table that did not exist.